charts/kubernetes-mcp-server/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "kubernetes-mcp-server.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "kubernetes-mcp-server.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- tpl (toYaml .) . | nindent 4 }}
+  {{- end }}
+{{- end }}
```

Comment:
@sabre1041 maybe I'm missing something but don't we need a `ClusterRole` + `ClusterRoleBinding` if `require_oauth` is not set to `true` (`false` is the default)?